### DISCUSSION TITLE
How do we let other containers talk to each other on the Glueun network?

### DISCUSSION THREAD

**Discussion by ghost** [qdm12/gluetun#1446] [Q&A]
I'm trying to allow my Radarr container access to my Transmission container so it can add torrents automatically to the Transmission container.

I can't get the Radarr container to see the Transmission container, and I don't know what to do. I can't get it to see the Transmission container via its IP address, hostname or container ID.

Does anyone know how to get this working?

**Comment by bnhf** [comment]
@ae5960e8-a6fc-491f-b252-898ecf59af95 

So assuming you're using a stack which includes Gluetun, Radarr and Transmission, all 3 containers share an IP -- so you'd reference them as localhost.
